src/App.styles.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";

```

Comment:
I will advise that you create a `.style.js` file per component. Not a global one